rust/opendp-ffi/src/trans/bootstrap.json

```diff
@@ -401,6 +401,18 @@
         ],
         "ret": {"c_type": "FfiResult<AnyTransformation *>"}
     },
+    "make_drop_null": {
+        "description": "Make a Transformation that drops null values.",
+        "features": ["contrib"],
+        "args": [
+            {
+                "name": "DA",
+                "is_type": true,
+                "description": "domain of data being imputed. This is OptionNullDomain<AllDomain<TA>> or InherentNullDomain<AllDomain<TA>>"
```

Comment:
Copied description from make_impute?

Reply:
```suggestion
                "description": "atomic domain of input data that contains nulls. This is OptionNullDomain<AllDomain<TA>> or InherentNullDomain<AllDomain<TA>>"
```

Reply:
It's cool that you can suggest an edit and then apply it directly from the PR UI.